TAF/utils/scripts/docker/get-compose-file.sh

```diff
@@ -41,7 +41,9 @@ for compose in ${COMPOSE_FILE}; do
 
   sed -i '/PROFILE_VOLUME_PLACE_HOLDER: {}/d' ${compose}.yml
   sed -i 's/\EXPORT_HOST_PLACE_HOLDER/${DOCKER_HOST_IP}/g' ${compose}.yml
-  sed -i 's/\MQTT_BROKER_ADDRESS_PLACE_HOLDER/${MQTT_BROKER_HOSTNAME}/g' ${compose}.yml
+  sed -i 's/TRIGGER_EXTERNALMQTT_URL: MQTT_BROKER_ADDRESS_PLACE_HOLDER/TRIGGER_EXTERNALMQTT_URL: tcp:\/\/${MQTT_BROKER_HOSTNAME}:1883/g' ${compose}.yml
```

Comment:
The trigger should come from the external broker
```suggestion
  sed -i 's/TRIGGER_EXTERNALMQTT_URL: MQTT_BROKER_ADDRESS_PLACE_HOLDER/TRIGGER_EXTERNALMQTT_URL: tcp:\/\/${EXTERNAL_BROKER_HOSTNAME}:1883/g' ${compose}.yml
```

Reply:
Updated. Replace all `MQTT_BROKER_ADDRESS_PLACE_HOLDER` to `tcp:\/\/${EXTERNAL_BROKER_HOSTNAME}:1883`